gen/calc.py

```diff
@@ -335,7 +335,7 @@ def validate_cluster_packages(cluster_packages):
 
 def calculate_no_proxy(no_proxy):
     user_proxy_config = validate_json_list(no_proxy)
-    return ",".join(['*.mesos,127.0.0.1,localhost'] + user_proxy_config)
```

Comment:
I assume this was not causing any bug right? Because it is not "just asterisk", It is `*.mesos` (presumably used with the understanding every host in the .mesos domain).